### DISCUSSION TITLE
Eigenes "Sicherheitstacho"

### DISCUSSION THREAD

**Discussion by 911mikelee** [telekom-security/tpotce#756] [Q&A] [answered]
Hallo Zusammen

Wenn mich nicht alles täuscht, basiert die Website https://www.sicherheitstacho.eu/ auf diesem Github Projekt und stellt auf eine sehr coole Art die laufenden Attacken dar. Wir möchten gerne ebenfalls einen Pot aufbauen und Daten zu laufenden Attacken beisteuern. Jedoch möchten wir die Attacken analog der Website sicherheitstacho.eu mit der Weltkarte (Map) auf der eigenen Website einbinden und die Attacken die weltweit laufen darstellen. 

Ist dies möglich?
Besten Dank.

**Comment by t3chn0m4g3** [comment, marked as answer]
We have no immediate plans to open source the code for Sicherheitstacho.
